.gitignore

```diff
@@ -40,3 +40,4 @@ tags
 
 course-data/*
 public/specs/*
+secrets.js
```

Comment:
I'd prefer we use env variables as opposed to a secrets file to store our secrets.